docs/BreakingChanges/147 BinaryFormatter can fail to find type from LoadFrom context.md

```diff
@@ -0,0 +1,44 @@
+## 147: BinaryFormatter can fail to find type from LoadFrom context
+
+### Scope
+Edge
+
+### Version Introduced
+4.5
+
+### Source Analyzer Status
+Planned
+
+### Change Description
+As of .NET Framework 4.5, a number of XmlSerializer changes may cause differences in deserialization when using `BinaryFormatter` to deserialize types that had been loaded in the
```

Comment:
Does it make sense now? The only instance we've seen is when XmlSerializer is used in conjunction with BinarySerializer. Seems like things get loaded into a different context or with different timing than in earlier versions.